frontend/src/apis/users/UserAPI.tsx

```diff
@@ -46,6 +64,24 @@ export interface UserTypeResp {
 }
 
 export const USER_API = {
+  getUsers: async function (req: GetUsersReq): Promise<GetUsersResp> {
+    return await API.request({
+      url: USERS_EXTENSION,
+      method: METHOD_GET,
+      params: req,
+    });
+  },
+
+  getUsersNoPagination: async function (
```

Comment:
Since you updated everything else to have no pagination, should probably change this one too

Reply:
If you want, feel free to wait until the no pagination changes are merged into main, or just merge this branch with that one